script/setup.js

```diff
@@ -497,30 +499,37 @@ try {
 		successSpinnerBlock(`Finished API hydration.`);
 	}
 
-	await withSpinner(
-		async () => {
-			await fs.rm("./script", { force: true, recursive: true });
-			await fs.rm(".github/workflows/setup.yml");
-		},
-		{
-			startText: `Removing setup script...`,
-			successText: `Removed setup script.`,
-			stopText: `Error removing setup script.`,
-			errorText: `Could not remove setup script. `,
-		}
-	);
+	if (skipUninstalls) {
+		console.log();
+		console.log(
+			chalk.gray`➖ Skipping removal of devDependencies only used for setup.`
```

Comment:
![image](https://user-images.githubusercontent.com/3335181/230631755-c945294e-b6d8-449d-80e5-1b039958fe14.png)

```plaintext
│
◇  ✅ Generated all-contributors table in README.md.
│
◇  ✅ Deleted local git tags.
│
◇  ➖ Skipping API hydration.

➖ Skipping removal of devDependencies only used for setup.
│
└  Great, looks like everything worked! 🎉
```

I like the gray of this text but the indentation of the previous one... will unify myself.